src/screens/ChannelListScreen/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-use-before-define */
+import * as React from 'react';
+import {FlatList} from 'react-native';
+
+import CommunityChannelItem from '../../components/ChatList/CommunityChannelItem';
+import GroupChatChannelItem from '../../components/ChatList/GroupChatChannelItem';
+import MessageChannelItem from '../../components/AnonymousChat/MessageChannelItem';
+import PostNotificationChannelItem from '../../components/AnonymousChat/PostNotificationChannelItem';
+import useFollowUser from './hooks/useFollowUser';
+import useSignedChannelListScreenHook from '../../hooks/screen/useSignedChannelListHook';
+
+const ChannelListScreen = () => {
+  const {channels, goToChatScreen, goToPostDetailScreen, goToCommunityScreen} =
+    useSignedChannelListScreenHook();
+  const {handleFollow, isInitialFollowing} = useFollowUser();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The destructuring assignment here makes the code less readable. Consider assigning each variable on a new line for better readability.

```diff
- const {channels, goToChatScreen, goToPostDetailScreen, goToCommunityScreen} = useSignedChannelListScreenHook();
- const {handleFollow, isInitialFollowing} = useFollowUser();
+ const hook = useSignedChannelListScreenHook();
+ const channels = hook.channels;
+ const goToChatScreen = hook.goToChatScreen;
+ const goToPostDetailScreen = hook.goToPostDetailScreen;
+ const goToCommunityScreen = hook.goToCommunityScreen;

+ const followUserHook = useFollowUser();
+ const handleFollow = followUserHook.handleFollow;
+ const isInitialFollowing = followUserHook.isInitialFollowing;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->